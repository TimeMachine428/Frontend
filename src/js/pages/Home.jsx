import React from "react";


export default class Home extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      showModal: false,
      username: "",
      password: "",
      isLoggedIn: false
    };
  }
  render() {
    return (
      <div>
        <h1>
        	Home
        </h1>
          <p>This page is visible to everyone</p>
      </div>
    );
  }
}