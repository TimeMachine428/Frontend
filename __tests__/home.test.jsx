import React from 'react';
import Home from '../src/js/pages/Home.jsx';
import renderer from 'react-test-renderer';

test('Home page information correct',() =>{
    const component = renderer.create(
        <Home />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})

