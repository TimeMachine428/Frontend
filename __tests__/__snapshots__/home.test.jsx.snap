// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page information correct 1`] = `
<div>
  <h1>
    Home
  </h1>
  <p>
    This page is visible to everyone
  </p>
</div>
`;
